tests/Rector/Property/DoctrineTargetEntityStringToClassConstantRector/config/configured_rule.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+declare(strict_types=1);
+
+use Rector\Config\RectorConfig;
+use Rector\Doctrine\Rector\Property\DoctrineTargetEntityStringToClassConstantRector;
+
+return static function (RectorConfig $rectorConfig): void {
+    $rectorConfig->rule(DoctrineTargetEntityStringToClassConstantRector::class);
```

Comment:
The package config must be included here too. See other rules:

https://github.com/rectorphp/rector-doctrine/blob/77c17ba507dc8502922ed36e7647532267963e65/tests/Rector/Class_/BlameableBehaviorRector/config/configured_rule.php#L10